### PR TITLE
SRVCOM-1728: Updating broker and trigger abstracts for Jupiter guidelines

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for 4.6+

Jira: https://issues.redhat.com/browse/SRVCOM-1728

Direct preview links:
- https://deploy-preview-44475--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-using-brokers.html
- https://deploy-preview-44475--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-triggers.html

### CODE REVIEW

**Review by jrangelramos — COMMENTED** [review]
QE Approved

File: modules/kn-trigger-list.adoc [hunk 1]
```diff
@@ -6,12 +6,12 @@
 [id="kn-trigger-list_{context}"]
 = Listing triggers by using the Knative CLI
 
-The `kn trigger list` command prints a list of available triggers.
+Using the `kn` CLI to list triggers provides a streamlined and intuitive user interface. You can use the `kn trigger list` command to list existing triggers in your cluster by using the `kn` CLI.
```
Comment (nalhadef) [inline modules/kn-trigger-list.adoc:9]: These two sentences seem to say the same thing.

**Review by nalhadef — COMMENTED** [review]
A single comment, otherwise, LGTM!

File: modules/kn-trigger-list.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/kn-trigger-list.adoc:9]: tweaked a bit to make it less repetitve, thanks

**Pull request merged**